src/main/java/htsjdk/samtools/util/AbstractProgressLogger.java

```diff
@@ -60,10 +61,18 @@ private synchronized void record() {
         if (this.lastChrom == null) readInfo = "*/*";
         else readInfo = this.lastChrom + ":" + fmt.format(this.lastPos);
 
+        final String rnInfo;
```

Comment:
Why not use `readName` instead of `rn` here? That would match what the rest of the code here does. Also, I believe that abbreviations like this make the code harder to read.